[docs] FAQ: How do I customize messages on Novu? (#1838)
novuhq/novu: docs/docs/community/faq.md

- MongoDB
- Redis
- File storage (S3/GCP/Azure) - Optional

## How do I customize messages on Novu?

In Novu, [handlebar variables](https://handlebarsjs.com/guide/) (variables enclosed within double curly brackets) are used to customize messages. Using this, one can take advantage of the following tools:

> - **Custom Variables:** You can create [custom variables](https://docs.novu.co/platform/templates#variable-usage) by defining it in the payload and access the value of the same using the following syntax:

Reviewer: ```suggestion
- **Custom Variables:** You can create [custom variables](https://docs.novu.co/platform/templates#variable-usage) by defining it in the payload and accessing its value using the following syntax:
```

What do you think? Something feels off to me in the sentence 'the value of the same'🤔

Author: Yeah, this sounds a lot more natural! ✅

  ```
  {{variable_name}}
  ```

- **Iteration/Looping:** You can iterate through a list of variables using the [each](https://docs.novu.co/platform/templates#iteration) keyword. The code within the block will loop once for each element that is present in the iterable variable. The syntax is as follows:
  ```
  {{#each iterable}}
    ...
  {{/each}}
  ```

> - **Conditional Block**

Reviewer: ```suggestion
- **Conditional Block:**
```
